code/modules/client/client_procs.dm

```diff
@@ -876,9 +876,9 @@
 	winset(src, "rpane.forumb", "background-color=#40628a;text-color=#FFFFFF")
 	winset(src, "rpane.rulesb", "background-color=#40628a;text-color=#FFFFFF")
 	winset(src, "rpane.githubb", "background-color=#40628a;text-color=#FFFFFF")
-	/* Mainwindow */
-	winset(src, "mainwindow.saybutton", "background-color=#40628a;text-color=#FFFFFF")
-	winset(src, "mainwindow.mebutton", "background-color=#40628a;text-color=#FFFFFF")
+	/* Outputwindow */
+	winset(src, "outputwindow.saybutton", "background-color=#40628a;text-color=#FFFFFF")
+	winset(src, "outputwindow.mebutton", "background-color=#40628a;text-color=#FFFFFF")
```

Comment:
Not really familiar with this sort of stuff, what does this change do?

Reply:
Since we've moved the say and me buttons to the output window, they are now a sub-component of that.